src/app/home/home.page.ts

```diff
@@ -98,7 +95,7 @@ export class HomePage {
     this.loadingItem = null;
   }
 
-  private addToFavorites(item: Country): void {
+  private addToFavorites(item: Country): void {  
     if (!this.favourites.includes(item))
```

Comment:
Using  Set is a built-in JavaScript object that stores unique values of any type, whether primitive values or object references The add method of Set automatically ensures that duplicate items are not added